src/controllers/workspace.controller.js

```diff
@@ -50,26 +50,14 @@ exports.updateWorkspace = async (req, res) => {
     try {
         const { name, description, workspaceId } = req.body;
 
-        if (
-            !name ||
-            !description ||
-            !workspaceId 
-        ) {
+        if (!name && !description && !workspaceId  ) 
+        {
             return response_400(
                 res,
-                "Invalid Request: Missing required fields"
+                "Invalid Request: No field to update"
             );
         }
 
-        const workspaceIdExists = await Workspace.findOne({
-            workspaceId: workspaceId,
-        }).exec();
-        if (workspaceIdExists) {
-            return response_400(
-                res,
-                "Invalid Request: workspaceId already in use"
-            );
-        }
```

Comment:
This check is necessary otherwise there will be two workspace with same workspaceId

Reply:
but we are updating not creating. We are not creating a new entity in database. We are simply updating the existent one.(which  is  being done in middleware)

Reply:
I mean  the check that workspace exists